packages/manager/modules/sms/src/sms/dashboard/creditTransfer/creditTransfer.controller.js

```diff
@@ -0,0 +1,81 @@
+export default class SmsCreditTransferCtrl {
+  /* @ngInject */
+  constructor($translate, SmsService, atInternet) {
+    this.$translate = $translate;
+    this.smsService = SmsService;
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.loading = true;
+    this.model = {
+      serviceName: null,
+      smsAccountTarget: null,
+      credits: null,
+    };
+    this.maxCredit = null;
+
+    return this.smsService
+      .getAccountList()
+      .then((accounts) => {
+        this.accountsList = accounts.map((account) => ({
+          ...account,
+          displayName:
+            account.description !== ''
+              ? `${account.name} (${account.description})`
+              : account.name,
+        }));
+
+        this.model.serviceName = this.accountsList.find(
+          (account) => account.name === this.serviceName,
+        );
```

Comment:
**question:** If I understand well, `this.model.serviceName` has the accountInfo not just serviceName. If so, can you please rename the variable?